backend/controller/controller.go

```diff
@@ -201,8 +204,9 @@ func (s *Service) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	creq := connect.NewRequest(&ftlv1.CallRequest{
-		Verb: &schemapb.VerbRef{Module: route.Module, Name: route.Verb},
-		Body: body,
+		Metadata: &ftlv1.Metadata{},
```

Comment:
I also might be able to add it to the `Metadata` here or something.

Reply:
coolio! Maybe we can chat today on this one. Want to make sure we have request validation in the right place. For example, if we add a `RequestParams` arg that has either the path params or query params here, when we create the actual verb request, we'll need a way to know if we should use the RequestParams for the `<verb>Request`.

Reply:
Off the top of my head, I think it should be in the schema validator.